sample-config/compose/plane.yml

```diff
@@ -0,0 +1,84 @@
+version: "3.9"
+services:
+  drone:
+    container_name: plane-drone
+    image: ghcr.io/drifting-in-space/plane-drone:latest
+    build:
+      dockerfile: ./drone/Dockerfile
+      context: ../..
+    volumes:
+      - ../auth:/etc/plane/auth:ro
+      - ../plane-config/drone.toml:/etc/plane/drone.toml:ro
+      - /var/run/docker.sock:/var/run/docker.sock
+    command:
+      /etc/plane/drone.toml
+    networks:
+      - plane
+
+  controller:
+    container_name: plane-controller
+    image: ghcr.io/drifting-in-space/plane-controller:latest
+    volumes:
+      - ../auth:/etc/plane/auth:ro
+      - ../plane-config/controller.toml:/etc/plane/controller.toml:ro
+    command:
+      /etc/plane/controller.toml
+    networks:
+      plane:
+        ipv4_address: 172.16.238.11
+
+  nats:
+    container_name: plane-nats
+    image: nats:latest
+    command:
+      "-DV --jetstream"
```

Comment:
`-DV` is super chatty and makes it impossible to read anything else in the logs, let's not use it as the default.